src/page-support/exclusive-content/components/dependent-script.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import SyntaxHighlighter from 'react-syntax-highlighter'
+import { docco } from 'react-syntax-highlighter/dist/esm/styles/hljs'
+import { CopyContent } from './copy-content'
+
+import { useExclusiveContent } from '../state'
+
+export function DependentScript() {
+  const [exclusiveContent] = useExclusiveContent()
+  const proxyPaymentPointer = exclusiveContent.verifier.endsWith('/')
+    ? exclusiveContent.verifier
+    : exclusiveContent.verifier + '/'
+  // TODO: change url to exclusive content script
+  const dependentScript = `<script type="module">
+  import { exclusiveContent } from "./exclusive-content.js";
+
+  var data = {
+    paymentPointer: "${exclusiveContent.pointer}",
```

Comment:
we shouldn't interpolate into js directly without some thorough escaping.

Reply:
I'm not sure I know what you would like to escape. Can there be double quotes in the payment pointer? What I could and probably should do is verify the pointer using the [method you already wrote](https://github.com/WICG/webmonetization/blob/4158435b2fb53dab8e7a3733e73dd7a8bc8f8c83/src/page-support/prob-revshare/lib/index.js#L167).

Reply:
In general:
JSON.stringify is handy to serialize strings for code generation